packages/autocannon-ui-frontend/src/components/App.jsx

```diff
@@ -33,7 +33,9 @@ const useStyles = makeStyles(theme => ({
     marginBottom: theme.spacing(2)
   },
   resultsGridItem: {
-    marginTop: theme.spacing(2)
+    marginTop: theme.spacing(2),
+    border: 'solid #d6d6d6 1px',
```

Comment:
can we get this color from the theme instead? it's a good practice to avoid hardcoded colors and use theme colors instead

Reply:
I agree wtih this, I will investigate

Reply:
Done